client/my-sites/post-share/style.scss

```diff
@@ -66,66 +65,28 @@ $section-border: solid 1px transparentize( lighten( $gray, 20% ), .5 );
 }
 
 .post-share__form {
-	flex: 3;
```

Comment:
<img src="https://cloud.githubusercontent.com/assets/77539/26261943/8be3a26a-3ca9-11e7-918b-b8b696e0c035.png" width="300px" />

I see a different width between the forms. If we need to apply different styles I'd consider creating a new component into `<PostShare />`, instead of reusing the same in the post editor sidebar and here.